### PR TITLE
#41 and #949. Unit test to compare the difference between System.Rune and System.Text.Rune.

### PR DESCRIPTION
**Pull request by BDisp**

### CODE REVIEW

File: UnitTests/TextFormatterTests.cs [hunk 1]
```diff
@@ -2242,5 +2242,69 @@ public void Reformat_Unicode_Wrap_Spaces_NewLines ()
...
+			Assert.Equal (1, Rune.ColumnWidth (c));
+			Assert.Equal (1, ustring.Make (c).ConsoleWidth);
+			Assert.Equal (1, ustring.Make (c).Length);
+
+			c = new System.Rune (123);
+			Assert.Equal (1, Rune.ColumnWidth (c));
+			Assert.Equal (1, ustring.Make (c).ConsoleWidth);
+			Assert.Equal (1, ustring.Make (c).Length);
+
+			c = new System.Rune (4432);
```
Comment (tig) [inline UnitTests/TextFormatterTests.cs:2264]: Pls use `/u` notation.

**Comment by tig** [comment]
This may sound sacrilegious, but why do we use nstack at all?

**Comment by BDisp** [comment]
I don't know why it gave the error.

**Comment by BDisp** [comment]
> 
> 
> This may sound sacrilegious, but why do we use nstack at all?

Because of the `ustring `type that I think only exists in that library.

**Comment by tig** [comment]
> > This may sound sacrilegious, but why do we use nstack at all?
> 
> Because of the `ustring `type that I think only exists in that library.

I meant to ask, why are we even using `ustring`?

**Comment by BDisp** [comment]
> I meant to ask, why are we even using `ustring`?

I do not know why. It has always existed here. In fact, it was here that I learned of the existence of this type. Only @migueldeicaza can answer that.

**Comment by BDisp** [comment]
> 
> 
> > > This may sound sacrilegious, but why do we use nstack at all?
> > 
> > 
> > Because of the `ustring `type that I think only exists in that library.
> 
> I meant to ask, why are we even using `ustring`?

@tig I think @migueldeicaza will prefer to use `NStack `because `System.Text.Rune` is not available in `net472` and in `netstandard2.0`.

**Pull request merged**